### ISSUE TITLE
Android emulator build error

### ISSUE THREAD

**Issue by kosso** [appcelerator/vscode-appcelerator-titanium#41]
The extension fails to build on a Genymotion emulator. 

### Steps to reproduce the Bug
1. From the 'Titanium Explorer' within VS Code, select Android > Emulator > Then a Genymotion Emulator eg: "Google Nexus 5X ..."
2. Click the play button to build and deploy to the emulator. 

Build fails with `[ERROR] Invalid "--device-id" value "Google"` 

since the device-id is clearly incorrect. 

### Expected Behavior
I expect it to work without throwing an error. ) 

### Your Environment
macOs 10.13.6
VSCode 1.30.2
Ti SDK 7.5.0.GA (also 7.3.1.GA)

**Comment by kosso** [comment]
It appears that the build command simply needs quotes around the device name used in the `--device_id` switch.

**Comment by sandrolain** [comment]
Hello

I encounter the same problem, I report the command string that is executed, the quotes are missing.
Similar problem seems to be present even with some path to directories in case the name of a directory contains a space.

```
appc run --platform android --target emulator --device-id Samsung Galaxy S7 - 7.1.0 - API 25 - 1440x2560 --liveview
```

**Comment by ewanharris** [comment]
Sorry for the delay in fixing this folks! I've just published a new release (0.1.4) and it should now be live on the VS Code marketplace.

Thanks for the report.
